libs/datamodel/schema-ast/src/parser/datamodel.pest

```diff
@@ -96,6 +96,7 @@ doc_content = @{ (!NEWLINE ~ ANY)* }
 // ######################################
 maybe_empty_identifier = @{ ( "_" | "-" | ASCII_ALPHANUMERIC)* }
 non_empty_identifier = @{ ASCII_ALPHANUMERIC ~ maybe_empty_identifier }
+path_identifier = @{ ASCII_ALPHANUMERIC ~ maybe_empty_identifier ~ ("." ~ non_empty_identifier)* }
```

Comment:
test: I think this would match `a name. something` — there's a pest modifier for disallowing inner whitespaces (not sure but it could already be the `@{` that's already there.

Reply:
it is fine 🔥